tests/test-libstoreconsumer/main.cc

```diff
@@ -0,0 +1,34 @@
+#include "globals.hh"
+#include "store-api.hh"
+#include <iostream>
+
+using namespace nix;
+
+int main (int argc, char **argv)
+{
+    if (argc != 2) {
+        std::cerr << "Usage: " << argv[0] << " store/path/to/something.drv\n";
+        return 1;
+    }
+
+    std::string drvPath = argv[1];
+
+    initLibStore();
+
+    auto store = nix::openStore();
+
+    // build the derivation
+
+    std::vector<DerivedPath> paths {
+        DerivedPath::Built(
+            store->parseStorePath(drvPath),
+            OutputsSpec::Names{"out"}
+        )
+    };
+
+    store->buildPaths(paths, bmNormal, store);
+
+    // print the out path maybe?
```

Comment:
I gave this a try, but it's rather non-trivial and not actually necessary for this test.

Reply:
If you use `buildPathsWithResults` isn't it easy?

Reply:
Oh I skimmed over the `builtOutputs` field.
I guess I have to try again now. Undeniable, checked result is better...